doc/classes/LinkButton.xml

```diff
@@ -51,13 +56,22 @@
 			Set additional options for BiDi override.
 		</member>
 		<member name="text" type="String" setter="set_text" getter="get_text" default="&quot;&quot;">
-			The button's text that will be displayed inside the button's area.
+			Sets the label to the specified text.
 		</member>
 		<member name="text_direction" type="int" setter="set_text_direction" getter="get_text_direction" enum="Control.TextDirection" default="0">
 			Base text writing direction.
 		</member>
 		<member name="underline" type="int" setter="set_underline_mode" getter="get_underline_mode" enum="LinkButton.UnderlineMode" default="0">
-			Determines when to show the underline. See [enum UnderlineMode] for options.
+			Sets the underline mode. See [LinkButton.UnderlineMode] for the available modes.
+		</member>
+		<member name="url" type="String" setter="set_url" getter="get_url" default="">
+			Sets a URL for this [LinkButton]. If a URL has been set, it will be launched automatically with the default program of the operating system on click.
```

Comment:
```suggestion
			The URL for this [LinkButton]. If a URL has been set, it will be opened automatically with the operating system's default program assigned to the URL protocol when the button is pressed. For HTTP and HTTPS URLs, this is typically the user's default Web browser. See [method OS.shell_open] for more information.
```